src/components/App/App.jsx

```diff
@@ -55,6 +56,57 @@ class App extends React.Component {
     })
   }
 
+  getViewContents() {
+    switch (this.state.view) {
+      case 'tasks': return (
+        <TaskList
```

Comment:
Seeing all these components listed closely like this will help with refactoring later. I think it'd be nice to reduce the duplication between method handlers passed to each component, like addFilter and manageFilters.